packages/stream-metadata/src/routes/profileImage.ts

```diff
@@ -87,7 +88,7 @@ export async function fetchUserProfileImage(request: FastifyRequest, reply: Fast
 		}
 		const redirectUrl = `${config.streamMetadataBaseUrl}/media/${
 			profileImage.streamId
-		}?key=${bin_toHexString(key)}&iv=${bin_toHexString(iv)}`
+		}?key=${bin_toHexString(key)}&iv=${bin_toHexString(iv)}&size=${size}`
```

Comment:
this should be a new route not a query param, make sure you also update in harmony